2020/OSCON/innersource-open-inside-the-firewall/outline.md

```diff
@@ -0,0 +1,43 @@
+# Title
+
+InnerSource: Open Inside the Firewall
+
+# Abstract
+
+Open source helps create the world's most impactful software.
+It allows people from all walks to collaborate on projects that fill a common need.
+InnerSource applies the lessons learned from open source to company-internal software development.
+Sustained InnerSource practice unlocks developer creativity and efficiency.
+It reduces siloed, duplicated work and creates a dynamic, fulfilling work environment.
+
+This talk gives an introduction and overview of the breadth of current state of InnerSource in the industry.
+We'll discuss motivations for InnerSource, benefits seen, challenges faced, and areas of active investigation.
+This content comes not just from a single perspective,
+but from the breadth of experiences shared by dozens of software companies as told in the InnerSource Commons,
+an open organization dedicated to education and spread of InnerSource.
+
+No matter what your current interest or motivation for InnerSource, this presentation will give you the latest experiences that others in your place have already had.
+You'll be ready to build on their learnings.
+You'll also learn how to leverage the InnerSource Commons to stay in touch on an ongoing basis with the latest industry experiences in InnerSource.
+
+# Audience
+
+People that already believe that open source is the right way to develop software.
```

Comment:
I think the more useful audience to try and reach are people who _aren't_ bought into OSS e.g:
- People who are struggling to work on software across multiple teams/sites
- People who are fed up of their companies reinventing the wheel rather than sharing solutions across teams/departments.

Reply:
Sounds like both audiences would be worth captioning 
- the first ones for the differences that the application of OSS principles and practices within the firewall brings, e.g. the need to deal with the often more or less fully pre-planned time of people. 
- the second ones being the ones you brought up. 
My experience is that these are two groups often with very little to no overlap.

Aside of that: OSCON being OSCON - how much of a point would one have to assume that the conference topic itself leads to a higher presence of people from the first group?

Reply:
> the conference topic itself leads to a higher presence of people from the first group

That's what I was figuring.   These are already the believers that are attending the conference.

Reply:
A fair point, but then you're saying that your audience is 'everyone at OSCON'. I think you'll get more interest if you can target a specific sub-set of OSS believers.

Reply:
From an OSS believer angle, maybe:
- Open Source advocates who want a gentler way to push their companies into Open Source practices.

Reply:
Sounds great.